pkg/tfgen/installation_docs.go

```diff
@@ -23,11 +23,15 @@ func plainDocsParser(docFile *DocFile, g *Generator) ([]byte, error) {
 
 	//TODO: See https://github.com/pulumi/pulumi-terraform-bridge/issues/2078
 	// - translate code blocks with code choosers
-	// - apply default edit rules
 	// - reformat TF names
-	// - Translation for certain headers such as "Arguments Reference" or "Configuration block"
 	// - Ability to omit irrelevant sections
-	return []byte(contentStr), nil
+
+	// Apply edit rules to transform the doc for Pulumi-ready presentation
+	contentBytes, err := applyEditRules([]byte(contentStr), docFile)
+	if err != nil {
+		return nil, err
+	}
+	return contentBytes, nil
```

Comment:
```suggestion
	return applyEditRules([]byte(contentStr), docFile)
```

Reply:
in next PRs I'll be adding more here so I'll need the `contentBytes` for the next transformation 😅